cirq-core/cirq/ops/clifford_gate.py

```diff
@@ -433,7 +433,7 @@ def __pow__(self, exponent: float) -> 'CliffordGate':
         return CliffordGate.from_clifford_tableau(base_tableau)
 
     def __repr__(self) -> str:
-        return f"Clifford Gate with Tableau:\n {self.clifford_tableau._str_full_()}"
+        return f"Clifford Gate with Tableau:\n{self.clifford_tableau._str_full_()}"
```

Comment:
Is there a test for this?  Should we add one?

Reply:
+1 for adding a test, the `__repr__` code is not covered from `clifford_gate_test.py` (I guess the coverage check found some indirect calls).
```
$ pytest --cov=cirq.ops.clifford_gate --cov-report=term-missing cirq-core/cirq/ops/clifford_gate_test.py

Name                                  Stmts   Miss  Cover   Missing
-------------------------------------------------------------------
cirq-core/cirq/ops/clifford_gate.py     443     11    98%   134-140, 405, 409, 411, 436, 448, 477
-------------------------------------------------------------------
TOTAL                                   443     11    98%
```

Reply:
Good points, done adding unit tests for `CliffordGate.__repr__`